docs/spec/distribution/overview.md

```diff
@@ -0,0 +1,50 @@
+# Distribution
+
+## Overview
+
+Collected fees are pooled globally and divided out passively to validators and
```

Comment:
Why passively? (e.g. "Fee distribution is lazy so that we don't need to modify every account which will receive fees every block")

Reply:
yes passively meaning that the computations take place at withdrawal (and less calculations required)

Reply:
Right, I think we should include this rationale, since it might not be initially obvious to someone who hasn't thought through fee distribution to the extent we have.